examples/backgroundlayer.js

```diff
@@ -18,33 +18,29 @@ app.module = angular.module('app', ['ngeo']);
 
 
 /**
- * The application-specific background layer directive.
+ * The application-specific background layer component.
  *
- * The directive is based on Angular's select, ngOptions, ngModel, and
- * ngChange directives. ngChange is used to avoid adding a watcher on
+ * The component is based on Angular's select, ngOptions, ngModel, and
+ * ngChange components. ngChange is used to avoid adding a watcher on
  * the ngModel expression.
  *
  * Note: we don't need two-way binding for ngModel here, but using ::
  * for the ngModel expression doesn't actually make a difference. This
  * is because ngModel doesn't actually watch the ngModel expression.
  *
- * @return {angular.Directive} Directive Definition Object.
- * @ngInject
+ * @type {!angular.Component}
  */
-app.backgroundlayerDirective = function() {
-  return {
-    restrict: 'E',
-    scope: {
-      'map': '=appBackgroundlayerMap'
-    },
-    templateUrl: 'partials/backgroundlayer.html',
-    bindToController: true,
-    controller: 'AppBackgroundlayerController as ctrl'
-  };
+app.backgroundlayerComponent = {
+  bindings: {
+    'map': '=appBackgroundlayerMap'
+  },
+  templateUrl: 'partials/backgroundlayer.html',
+  controller: 'AppBackgroundlayerController',
+  controllerAs: 'ctrl'
```

Comment:
Can't we remove that ? Then use "$ctrl" in the partial ?

Reply:
yes, but then we need to update partials, I am not sure that it is critical and/or should be done in that PR

Reply:
That's not critical. That's why we'll probably never do that if we don't do that right now.
But it's not really important. If you think it takes too much time to do the change (and test) leave it like that :-)

Reply:
it's fine to leave as it for the examples but it would be nice to do it the all the ngeo and gmf components (in the others pull requests).

Reply:
I will do it as another PR on monday